cfgov/login/signals.py

```diff
@@ -15,7 +13,3 @@ def user_save_callback(sender, **kwargs):
         PasswordHistoryItem.objects.create(
             user=user, encrypted_password=user.password
         )
-
```

Comment:
I think this whole `user_save_callback` and in fact this whole file can be deleted, no? This still refers to PasswordHistoryItem which has been deleted. Maybe this isn't triggering the linter because it's not imported from anywhere?

Reply:
Removed!